playbooks/roles/openvpn/templates/etc_openvpn_server_udp.conf.j2

```diff
@@ -0,0 +1,301 @@
+#################################################
```

Comment:
This is also a lot of duplication relative to `etc_openvpn_server.conf.j2`. It would be cool to pull all of the common lines into a shared template that the TCP and UDP server configuration templates could both include. That way they would only need to contain the lines that are different.